src/eddymotion/model/base.py

```diff
@@ -409,6 +412,119 @@ class DKIModel(BaseModel):
     _model_class = "dipy.reconst.dki.DiffusionKurtosisModel"
 
 
+class GaussianProcessModel:
+    """A Gaussian Process model based on [Andersson16a]_ (fig 1).
+    DWIs need to be transformed to a single ref space (fig 2 [Andersson16b]_ ?)
+
+    Definitions:
+    s: reference/undistorted space: used to denote the space or any image in
+    that space
+    f: observed/distorted image: used to denote any image in acquisition space
+    a: acquisition parameters: PE‐direction and bandwidth in PE‐direction
+    r: rigid body (subject movement) parameters
+    \beta: Eddy current parameters
+    e(\beta): Eddy current‐induced off resonance field (Hz)
+    h: Susceptibility induced off‐resonance field (Hz)
+
+    (fig 1) and algorithm:
+    1. Input: N DWI volumes f_{i} with acq parameters a_{i}; susceptibility
+    field h
+    2. Initialize: set all beta_{i} and r_i{i} = 0
```

Comment:
This is the estimator's job, not the model AFAICT
```suggestion
    2. Initialize: set all beta_{i} and r_i{i} (passed in by the estimator)
```

Reply:
Same comment as in https://github.com/nipreps/eddymotion/pull/188#discussion_r1632118544.